packages/visualizations/src/components/Map/utils.js

```diff
@@ -1,19 +1,40 @@
 import chroma from 'chroma-js';
 import geoViewport from '@mapbox/geo-viewport';
 
-export const colorShapes = (geoJson, values, colorScale) => {
+export const colorShapes = (geoJson, values, colorsScale) => {
     // Key in the values is "x"
     // Key in the shapes is "key"
     // We add a color property in the JSON
+    // FIXME Do we always want to take y or accept other keys
     const rawValues = values.map((v) => v.y);
     const min = Math.min(...rawValues);
     const max = Math.max(...rawValues);
-
-    // For now the colorscale is a single color, and we build a scale from it. TBD
-    const colorMin = chroma(colorScale).darken(4).hex();
-    const colorMax = chroma(colorScale).brighten(4).hex();
-
-    const scale = chroma.scale([colorMin, colorMax]).domain([min, max]);
+    let colorMin;
+    let colorMax;
+    let scale;
+
+    if (colorsScale?.type === 'palette') {
+        const tresholdArray = [];
```

Comment:
Spelled "threshold" (thanks google, I knew my brain was reading something weird but I wasn't able to spell the right letters)

Reply:
Good catch ! I changed it 👍